iac-extensions/arm/src/test/java/org/sonar/iac/arm/tree/api/ArrayExpressionAssert.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * SonarQube IaC Plugin
+ * Copyright (C) 2021-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.iac.arm.tree.api;
+
+import org.assertj.core.api.Assertions;
+
+import static org.sonar.iac.arm.tree.api.ExpressionAssert.assertThat;
+
+public class ArrayExpressionAssert extends HasTextRangeAssert<ArrayExpressionAssert, ArrayExpression> {
+  protected ArrayExpressionAssert(ArrayExpression arrayExpression) {
```

Comment:
The constructor can be private. The same in other classes (but without `HasTextRangeAssert`)

Reply:
There is `assertThat()` method for that, so the constructors can be private

Reply:
Make sense, will do that 👍